shell/platform/windows/public/flutter_windows.h

```diff
@@ -47,6 +47,14 @@ typedef struct {
   // it will be ignored in that case.
   const wchar_t* aot_library_path;
 
+  // The name of the top-level Dart entrypoint function. If null or the empty
+  // string, 'main' is assumed. If a custom entrypoint is used, this parameter
+  // must specifiy the name of a top-level function from the same Dart library
+  // that contains the app's main() function, which must be decorated with
```

Comment:
Super nit: "which must be decorated" seems like it refers to the `main` function, I think the previous "and must be" was a tad clearer. I don't feel strongly about this though as anyone messing with this API probably gets it 😀

Reply:
Done. Thanks!